### PR TITLE
Code Quality: Improved app startup routine

### PR DESCRIPTION
**Pull request by d2dyno1**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?

### CODE REVIEW

File: src/Files.Core/Services/IApplicationService.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+		/// <summary>
+		/// Gets the version of the app.
+		/// </summary>
+		Version AppVersion { get; }
+
+		/// <summary>
+		/// Gets the path at which the App Logo is located.
+		/// </summary>
+		[Obsolete("This is a bad way of accessing the logo. Use something more abstract instead, and ideally move it out of this interface.")]
+		string AppLogoPath { get; }
```
Comment (yaira2) [inline src/Files.Core/Services/IApplicationService.cs:22]: Can we rename to AppIcoPath? The current name has been confusing for some time.

File: src/Files.Core/Models/IDatabaseModel.cs
Comment (0x5bfa) [inline src/Files.Core/Models/IDatabaseModel.cs:1]: Can you move this into Core.Data?

File: src/Files.Core/Models/ISerializedModel.cs
Comment (0x5bfa) [inline src/Files.Core/Models/ISerializedModel.cs:1]: As above

File: src/Files.Core/AppModels/BaseDatabaseModel.cs
Comment (0x5bfa) [inline src/Files.Core/AppModels/BaseDatabaseModel.cs:1]: As above

File: src/Files.Core/AppModels/SingleFileDatabaseModel.cs
Comment (0x5bfa) [inline src/Files.Core/AppModels/SingleFileDatabaseModel.cs:1]: As above

**Review by 0x5bfa — APPROVED** [review]
namespace corrections.

**Comment by d2dyno1** [comment]
This is Phase 1 of startup routine improvements.
(More are on their way but will be in separate PRs to avoid conflicts)

**Review by yaira2 — APPROVED** [review]

File: src/Files.App/App.xaml.cs [hunk 2]
```diff
@@ -20,32 +18,21 @@
...
 using Files.Core.Services.SizeProvider;
 using Files.Core.Storage;
 using Files.Shared;
 using Files.Shared.Cloud;
-using Files.Shared.Extensions;
 using Files.Shared.Services;
 using Files.Shared.Services.DateTimeFormatter;
-using Microsoft.AppCenter;
-using Microsoft.AppCenter.Analytics;
-using Microsoft.AppCenter.Crashes;
```
Comment (yaira2) [inline src/Files.App/App.xaml.cs:35]: This is still needed for AppCenter.

**Comment by d2dyno1** [comment]
I'll investigate

**Pull request merged**